examples/adhoc.yaml

```diff
@@ -0,0 +1,10 @@
+network:
```

Comment:
Perhaps this file could be called wireless_adhoc.yaml? It would be easier to find it if someone is looking for an example of ad-hoc wifi configuration in the examples/ folder.

Also, having an IP address and a default route would be good, people would be able to just copy, paste, make minimal changes to it and have the interface working :)